components/navbar/Navbar.js

```diff
@@ -38,9 +38,9 @@ export default function Navbar() {
     if (currentTheme === "dark") {
       return (
         <button className="p-2 text-white" onClick={() => setTheme("light")}>
-          <FaSun />
+          <FaSun className="text-gray-400 hover:text-white"/>
```

Comment:
We are trying to not use `...gray-400`, but our predefined colors in the tailwind config

Reply:
I noticed, in dark mode we have have no borders around the buttons, so I will fix that and update the `-400`